troposphere/static/js/components/modals/instance/launch/steps/BasicLaunchStep.react.js

```diff
@@ -15,43 +15,15 @@ export default React.createClass({
                     <div className="col-md-6">
                         <h3 className="t-title">Basic Info</h3>
                         <hr/>
-                        <BasicInfoForm
-                            image={this.props.image}
-                            project={this.props.project}
-                            projectList={this.props.projectList}
-                            imageVersionList={this.props.imageVersionList}
-                            imageVersion={this.props.imageVersion}
-                            onNameChange={this.props.onNameChange}
-                            onVersionChange={this.props.onVersionChange}
-                            onProjectChange={this.props.onProjectChange}
-                        />
+                        { React.createElement(BasicInfoForm, this.props) }
```

Comment:
`BasicLaunchStep` is an inbetween component, stuff gets passed through it from parent to child. If you have something reading state in the middle of the pipe, make it as dumb as possible, otherwise its too easy to forget to include something, etc.